network/load-balancer/reference-content/configuring-load-balancer.mdx

```diff
@@ -0,0 +1,137 @@
+
+## Configuring a backend 
+
+The configuration that you choose for your Load Balancer's backend determines the manner in which it transfers requests to its associated backend servers.
+
+A number of configuration settings are available.
+
+<Message type="note">
+
+  Here, the term "backend" refers to a set of configurations that determine how the Load Balancer forwards traffic. You can create one or more backends for your Load Balancer. The term "backend server" designates the final-destination backend servers which the Load Balancer forwards traffic on to. Each backend must specify one or more backend servers (via their IP addresses) to forward to.
+
+</Message>
+
+### Choosing a protocol and port 
+
+The Load Balancer initiates connections to its backend servers using the protocol and port you define. 
+
+* **HTTP protocol (typically port 80)**: The Load Balancer will initiate an HTTP connection with the backend server. Traffic will be forwarded without encryption.
+* **HTTPS protocol (typically port 443)**: The Load Balancer will attempt to initiate an HTTPS connection with the backend server. In this case, the backend server must have an SSL/TLS certificate. Traffic will be forwarded using HTTPS, i.e. with encryption.
+* **TCP protocol**: The Load Balancer will attempt to initiate a TCP (layer 4) connection to the backend server. 
+
+### Choosing a Verify Certificate setting
+
+If you select the HTTPS protocol, you are prompted to choose a Verify Certificate setting. Note that this setting does not display if you selected HTTP or TCP protocol.
+
+When the Load Balancer initiates an HTTPS connection to a backend server, the backend server presents its certificate. 
+- When Verify Certificate is selected, the Load Balancer checks the backend server's certificate and only connects if it is valid.
+- When Verify Certificate is unselected, the Load Balancer does not check the backend server's certificate, connecting to the backend server regardless of the certificate's validity. 
+
+This setting mostly concerns those using self-signed certificates on the backend server/s.
+
+Note that the same Verify Certificate setting you select here will also be used for health checks, if you choose HTTPS for your [health check method](#choosing-a-health-check-method).
+
+### Choosing a proxy protocol
+
+Proxy protocol determines how connection information from the client (eg the client’s IP address) will be transferred through the Load Balancer and on to the destination backend server. This allows the backend server to know the client's address, whatever the upper layer protocol (HTTP/HTTPS/TCP). 
+
+* **No proxy protocol**:  Disable proxy protocol.
+* **Proxy protocol v1**: Version one (text format).
+* **Proxy protocol v2**: Version two (binary format).
+* **Proxy protocol v2-ssl**: Version two with SSL connection
```

Comment:
In this case we use proxy protocol v2 and we add the SSL information extension which provides information on SSL connection settings originally sent by the client.

Reply:
done